package.json

```diff
@@ -58,7 +58,7 @@
     "test:js": "mocha --require src/lib/test-setup.js --recursive \"src/**/*-test.js\"",
     "deploy": "gh-pages -x -d build/web -r \"https://$GH_TOKEN@github.com/mozilla/FirefoxColor.git\"",
     "release:base": "npm-run-all clean build:web sign && mv addon.xpi build/web && npm run deploy",
-    "release:dev": "cross-env ADDON_URL='https://addons-dev.allizom.org/firefox/addon/firefox-color/' SITE_URL='https://mozilla.github.io/FirefoxColor/' SITE_ID=github npm run release:base"
+    "xpi:dev": "cross-env SITE_URL='https://mozilla.github.io/FirefoxColor/' SITE_ID=github npm run package && mv addon.xpi build/web/firefox-color-dev-unsigned.xpi"
```

Comment:
Could you move this to the line before `xpi:stage`? A bit easier to maintain.

The `release:dev` line here can be restored (or remove it altogether with `release:base`, since both are not used any more). The README still mentions `release:dev`, which is not accurate any more. I can update the documentation if you aren't sure on what to put there.